packages/replay/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@replayio/replay",
-  "version": "0.1.0-alpha.7",
+  "version": "0.1.0-alpha.8",
```

Comment:
If you incremented this by hand it might be better to leave it and let `lerna` do its thing when it publishes (but I could be wrong).